### ISSUE TITLE
broken due to missing 'isarray'

### ISSUE THREAD

**Issue by srl295** [swagger-api/swagger-js#1553]
I'm guessing this is a polyfill injected by the build process?

```
$ cd /tmp/_
$ nvm i 14
Now using node v14.4.0 (npm v6.14.5)
$ npm i swagger-client
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.

> core-js@2.6.11 postinstall /private/tmp/_/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"
…
+ swagger-client@3.10.6
$ node -p "require('swagger-client')"
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'isarray'
Require stack:
- /private/tmp/_/node_modules/swagger-client/dist/index.js
- /private/tmp/_/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/private/tmp/_/node_modules/swagger-client/dist/index.js:3018:18)
    at __webpack_require__ (/private/tmp/_/node_modules/swagger-client/dist/index.js:21:30)
    at Object.<anonymous> (/private/tmp/_/node_modules/swagger-client/dist/index.js:1196:15)
    at Object.<anonymous> (/private/tmp/_/node_modules/swagger-client/dist/index.js:2974:30)
    at __webpack_require__ (/private/tmp/_/node_modules/swagger-client/dist/index.js:21:30)
    at Object.<anonymous> (/private/tmp/_/node_modules/swagger-client/dist/index.js:594:41) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/private/tmp/_/node_modules/swagger-client/dist/index.js',
    '/private/tmp/_/[eval]'
  ]
}
$ npm i isarray
+ isarray@2.0.5
$ node -p "require('swagger-client')"
[Function: Swagger] {
  http: [Function: http],
…
```

**Comment by char0n** [comment]
Thanks for reporting this! I'll investigate this and get back to you.

**Comment by char0n** [comment]
I can confirm, this happens regardless of node version. Investigating further...

**Comment by char0n** [comment]
Released in [v3.10.7](https://www.npmjs.com/package/swagger-client/v/3.10.7)

**Comment by srl295** [comment]
Thanks!
